src/components/field.js

```diff
@@ -1,22 +1,26 @@
 import React, {PropTypes} from 'react';
 import DefaultInputComponent from './input';
-import DefaultDisplay from './display';
+import DefaultDisplayComponent from './display';
 import DefaultSelectComponent from './select';
 import DefaultListComponent from './list';
+import DefaultTextComponent from './text';
 
+const FieldLabelValueComponent = ({label, ValueComponent}) => (
+    <div className='field'>
+        <div><b>{label}</b></div>
+        {ValueComponent}
+    </div>
+);
+FieldLabelValueComponent.displayName = 'FieldLabelValueComponent';
 
 
-function Field({multiple, list,fieldFor,fieldForLine,  ...otherProps}) {
-    otherProps.value = otherProps.rawInputValue; https://github.com/get-focus/focus-redux/issues/39 compatibility with focus components
-    const {DisplayComponent = DefaultDisplay, InputComponent = DefaultInputComponent, SelectComponent = DefaultSelectComponent, ListComponent = DefaultListComponent} = otherProps.metadata;
-    const renderConsult = () => ( list ?  <ListComponent fieldForLine={fieldForLine} values={otherProps.formattedInputValue} {...otherProps}/> : <DisplayComponent {...otherProps} /> );
-    const renderEdit = () => list ?  <ListComponent fieldForLine={fieldForLine} values={otherProps.formattedInputValue} {...otherProps}/> : (multiple ? <SelectComponent {...otherProps}/> : <InputComponent {...otherProps}/>);
-    return (
-        <div className='field'>
-            <div><b>{otherProps.name}</b></div>
-            {otherProps.editing ? renderEdit() : renderConsult()}
-        </div>
-    );
+function Field({textOnly, multiple, list, fieldFor, fieldForLine, ...otherProps}) {
+    otherProps.value = otherProps.rawInputValue; //https://github.com/get-focus/focus-redux/issues/39 compatibility with focus components
+    const {TextComponent = DefaultTextComponent, DisplayComponent = DefaultDisplayComponent, InputComponent = DefaultInputComponent, SelectComponent = DefaultSelectComponent, ListComponent = DefaultListComponent} = otherProps.metadata;
+    const renderConsult = () => (list ? <ListComponent fieldForLine={fieldForLine} values={otherProps.formattedInputValue} {...otherProps}/> : (textOnly ? <TextComponent {...otherProps} /> : <DisplayComponent {...otherProps} />));
```

Comment:
Not expecially a remark on your work but this field womponent starts to be a bit heavy.
Maybe a switch case or a `FieldReadOnlyComponent` and a `FieldEditComponent` or both would be great.
cc @Ephrame @TomGallon